lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -155,31 +155,26 @@ private FileBasedSystemAccessControl(
         ImmutableSet.Builder<AnyCatalogPermissionsRule> anyCatalogPermissionsRules = ImmutableSet.builder();
```

Comment:
Can you please improve commit message. The current one is very generic. Maybe `Leverage Optional::stream in FileBasedSystemAccessControl`?